### DISCUSSION TITLE
APISIX Standalone SSL

### DISCUSSION THREAD

**Discussion by singhjoga** [apache/apisix#5562] [Q&A] [answered]
Hi Guys,
I am using APISIX in standalone mode. Is it possible to provide SSL certificate and key from a file, rather than embedding the certificate content in the apisix.yaml file?

Best Regards,

Joga

**Comment by tokers** [comment, marked as answer]
> Hi Guys, I am using APISIX in standalone mode. Is it possible to provide SSL certificate and key from a file, rather than embedding the certificate content in the apisix.yaml file?
> 
> Best Regards,
> 
> Joga

Unfortunately it's not feasible, but you can make some templates and render the contents by a specific program.
